src/app/components/navigation/navigation.component.html

```diff
@@ -22,14 +22,8 @@
         </ul>
 
       <div class="lang">
-        <div class="d-none d-sm-block">
-          <ng-container  *ngFor="let itemLang of enabledLanguages; let i = index">
-            <a [lang]="itemLang" (click)="setLanguage(itemLang)" [title]="languages[itemLang]" [class.selected]="lang===itemLang">{{ itemLang | titlecase }}</a>
-            <ng-container *ngIf="i < enabledLanguages.length - 1; else endLangNav"> | </ng-container>
-          </ng-container>
-          <ng-template #endLangNav> &nbsp; </ng-template>
-        </div>
-        <div class="d-sm-none">
+        <span class="icon"><i class="fa fa-language" aria-hidden="true"></i></span>
```

Comment:
Could you put the icon in a label linked to the select and provide an alternative text (in a `span.sr-only` for example) and a `title`?

Reply:
Sure, I've updated the code to make the language selection accessible.